### DISCUSSION TITLE
Locked out of account despite correct steps

### DISCUSSION THREAD

**Discussion by alphanumeric1d** [community/community#43104] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

i succeed in the authentication, but Github fails to send an SMS for verification, proceeds to lock me out, and does not respond to tickets (3 days passed).

**Comment by jfullstackdev** [comment]
if you did not try Support, try that too

https://support.github.com/contact/account

**Comment by alphanumeric1d** [comment]
still not contacted by support (4 days)
